framework/src/mesh/MooseMesh.C

```diff
@@ -599,8 +599,8 @@ MooseMesh::nodeToElemMap()
     Threads::spin_mutex::scoped_lock lock(Threads::spin_mtx);
     if (!_node_to_elem_map_built)
     {
-      MeshBase::const_element_iterator       el  = getMesh().elements_begin();
-      const MeshBase::const_element_iterator end = getMesh().elements_end();
+      MeshBase::const_element_iterator       el  = getMesh().active_elements_begin();
+      const MeshBase::const_element_iterator end = getMesh().active_elements_end();
```

Comment:
Without this change, we hit an assert in NonlinearSystem::findImplicitGeometricCouplingEntries().  I think this is the right thing to do.  I would expect that nodeToElemMap() would only return active elements.